app/src/androidTest/java/com/github/h3lp3rs/h3lp/HelpPageActivityTest.kt

```diff
@@ -0,0 +1,127 @@
+package com.github.h3lp3rs.h3lp
+
+import android.Manifest
+import android.content.Intent
+import android.location.Location
+import android.os.Bundle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.Until
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+import com.github.h3lp3rs.h3lp.locationmanager.LocationManagerInterface
+import org.hamcrest.Matchers.containsString
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.anyOrNull
+import org.mockito.Mockito.`when` as When
+
+const val EPIPEN = "Epipen"
+// Current coordinates to mock a user
+const val CURRENT_LAT = 46.514
+const val CURRENT_LONG = 6.604
+
+// Destination coordinates
+const val DESTINATION_LAT = 46.519
+const val DESTINATION_LONG = 6.667
+
+// Walking time from the user to the destination according to the Google directions API
+const val TIME_TO_DESTINATION = "1 hour 19 mins"
+
+@RunWith(AndroidJUnit4::class)
+class HelpPageActivityTest {
+    private val locationManagerMock: LocationManagerInterface =
+        mock(LocationManagerInterface::class.java)
+    private val locationMock: Location = mock(Location::class.java)
+    private val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(
```

Comment:
Is this rule useful, I feel like an explicit launch is made in every test function